headroom-templates/haskell.mustache

```diff
@@ -0,0 +1,10 @@
+{- |
+Module                  : MODULE_NAME
```

Comment:
Maybe it should be in double brackets too? I am not an expert, but usually, in mustache all the variables should be in such brackets.

Reply:
@vrom911 Makes sense to me. It seems like a reasonable UI where all variables are put in `{{ }}` and some of the variables are just provided by `headroom`.

Reply:
I think we're getting to point **2)** from above - Problem is that currently _Headroom_ cannot extract any variable values from Haddock module header already existing in processed source code file. Now, the only source of variable values is either `variables` section in `.headroom.yaml` or `-v|--variable "KEY=VALUE"` command line argument. There is existing issue https://github.com/vaclavsvejcar/headroom/issues/25 that should make this possible. Simply said, _Headroom_ will contain custom code for every supported file type that will be able to extract some specific content from source code files for templates - such as to parse existing Haddock module header into some variable values for _Haskell_ source code files. This is becoming top priority for next release, as it bothers me too, I just haven't had enough time to think about it thoroughly and implement it.